next/components/ui/Listing/Listing.tsx

```diff
@@ -2,14 +2,17 @@ import { CallToAction, RowSubcategory } from '@components/ui'
 import MLink from '@modules/common/MLink'
 import ShowMoreLink from '@modules/common/ShowMoreLink'
 import cx from 'classnames'
+import { useTranslation } from 'next-i18next'
 import React from 'react'
 
+import { NavikronosChildren } from '../../../navikronos/types'
+
 export interface ListingProps {
   className?: string
   title?: string
   url?: string
   moreLinkTitle?: string
-  pages: { title: string; url: string; moreLinkTitle: string }[]
+  children: NavikronosChildren
```

Comment:
Maybe we should avoid using "reserved" `children` and use other name for this prop?